releasing/README.md

```diff
@@ -308,7 +315,28 @@ refreshMaster
 testKustomizeRepo
 ```
 
-### Publish Official Docker Image
+## Update example test target
+
+[Makefile]: https://github.com/kubernetes-sigs/kustomize/blob/master/Makefile
+
+Edit the `prow-presubmit-target` in the [Makefile]
+to test examples against your new release.
+
+```
+sed -i "" "s/LATEST_V4_RELEASE=.*/LATEST_V4_RELEASE=v4.3.0/" Makefile
+title="Test examples against latest release"
+createBranch updateProwExamplesTarget $title
+createPr
+```
+
+Wait for tests to pass, then merge the PR:
+```
+gh pr status  # rinse, repeat
+gh pr merge -m
+```
+
+
+## Publish Official Docker Image
 
 [k8s.io]: https://github.com/kubernetes/k8s.io
 [k8s-staging-kustomize]: https://pantheon.corp.google.com/gcr/images/k8s-staging-kustomize?project=k8s-staging-kustomize
```

Comment:
Let's change this link to https://console.cloud.google.com/gcr/images/k8s-staging-kustomize?project=k8s-staging-kustomize - `pantheon.corp` is a google-internal link, I believe changing it to `console.cloud` makes it an externally viewable link that you have permissions to view.